modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,86 @@
+= ownCloud on a QNAP NAS
+
+== Introduction
+
+If you have a QNAP NAS (Network Attached Storage) system, you can install an ownCloud docker image from the QNAP App Center on your NAS. The free-of-charge community edition can be used with up to five users and an unlimited number of guest users.
+
+In case you want to use Enterprise features, you can purchase an Enterprise license from the QNAP store. This also allows you to purchase licenses for more users.
+
+The QNAP ownCloud is designed to work out of the box. We strongly recommend not to change the database (PostgreSQL) or the Web server (Apache). All configuration should happen in the Web interface. However `occ` commands are availabe.
+
+== Preparation
+
+If you haven't done so already, perform the following steps on you QNAP system:
+
+. Create a storage pool in addition to the system volume. You can combine hard disks and solid state disks in RAID groups for flexible storage. This pool can be resized later if necessary.
+
+. Install container-station version 2.0 or higher from the QNAP Store in the AppCenter.
```

Comment:
Maybe a hint for what this is good for / needed...

Reply:
Added a few words (virtualization and Docker).